client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -1,7 +1,8 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
 import debounce from 'lodash/debounce';
 import { findDOMNode } from 'react-dom';
 import classNames from 'classnames';
```

Comment:
`import analytics from 'lib/analytics';` is actually an internal dependency, right?